.github/workflows/linux_meson.yml

```diff
@@ -53,5 +65,6 @@ jobs:
       env:
         TERM: xterm-256color
       run: |
+        export LD_LIBRARY_PATH="/usr/local/lib/"  # to find libopenblas.so.0
```

Comment:
Move this to the general `env:` stanza for this CI job (line 31), I think building should use it as well to detect OpenBLAS.